mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/196 - asset view

Closes #196 

To test:
- Run any scenario and open Asset View by clicking Asset button
- Make sure assets have the same information as in the original game
